pkg/storage/below_raft_protos_test.go

```diff
@@ -74,8 +85,6 @@ var belowRaftGoldenProtos = map[reflect.Type]fixture{
 	reflect.TypeOf(&raftpb.HardState{}): {
 		populatedConstructor: func(r *rand.Rand) protoutil.Message {
 			n := r.Uint64()
-			// NB: this would rot if HardState ever picked up more (relevant)
-			// fields, but that's unlikely.
 			return &raftpb.HardState{
```

Comment:
Try just adding `XXX_unrecognized: nil,` here and see if that allows you to remove the linter warning.

Reply:
yep, you are right, thanks! fixed that.